ui/page/about_page.go

```diff
@@ -164,6 +157,24 @@ func (pg *AboutPage) layoutRows(gtx layout.Context) layout.Dimensions {
 	})
 }
 
+func (pg *AboutPage) licenseRowLayout(gtx C, in layout.Inset) D {
+	return pg.licenseRow.Layout(gtx, func(gtx C) D {
+		return layout.Flex{}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return in.Layout(gtx, pg.license.Layout)
+			}),
+			layout.Flexed(1, func(gtx C) D {
+				return layout.E.Layout(gtx, func(gtx C) D {
+					return in.Layout(gtx, func(gtx C) D {
+						pg.chevronRightIcon.Color = pg.Theme.Color.Gray1
+						return pg.chevronRightIcon.Layout(gtx, values.MarginPadding20)
+					})
+				})
+			}),
+		)
+	})
+}
```

Comment:
make this an inline func() by moving it to the section it was used. 

It was only ever used within that section. I don't see the need for this. except there is a reason though kindly share